oak-run/src/main/groovy/org/apache/jackrabbit/oak/console/GroovyConsole.groovy

```diff
@@ -124,6 +124,7 @@ class GroovyConsole {
                 new CheckpointCommand(shell),
                 new LsCommand(shell),
                 new PnCommand(shell),
+                new RevisionCommand(shell),
```

Comment:
You might already be aware of this, but just to be sure : do we need a new RevisionCommand when we have the RevisionsCommand?

Reply:
The intention of this was to be able to run from the interactive console. We would probably don't this anymore, since that console is read-only anyway.